common/djangoapps/third_party_auth/models.py

```diff
@@ -225,7 +227,7 @@ def get_social_auth_uid(self, remote_id):
         return remote_id
 
     @classmethod
-    def get_register_form_data(cls, pipeline_kwargs):
+    def get_register_form_data(cls, pipeline_kwargs, **kwargs):
```

Comment:
Passing an instance as a keyword argument to a class method seems like a Bad Thing. If we need that data, make it an instance method (remove the `@classmethod` decorator and change the `cls` arg to `self`).

Unless I'm missing something here?